src/components/BookingModal/index.js

```diff
@@ -135,10 +144,11 @@ const BookingModal = props => {
           />
         )}
         {renderLegacyMessage()}
-        {renderBookingModalForm()}
+        {!isRejectedHoliday ? renderBookingForm() : renderRejectionForm()}
```

Comment:
try to keep this kind of logic positive, a bit easier for other devs to work out quickly `{isRejectedHoliday ? renderRejectionForm() : renderBookingForm()}`